content/en/docs/contribute/review/for-approvers.md

```diff
@@ -0,0 +1,233 @@
+`lifecycle/frozen` | An issue with this label will not become stale after 90 days of inactivity. A user manually adds this label to issues that need to remain open for much longer than 90 days, such as those with a `priority/important-longterm` label.
+{{< /table >}}
+
+## Handling special issue types
+
+SIG Docs encounters the following types of issues often enough to document how
+to handle them.
+
+### Duplicate issues
+
+If a single problem has one or more issues open for it, combine them into a single issue.
+You should decide which issue to keep open (or
+open a new issue), then move over all relevant information and link related issues.
+Finally, label all other issues that describe the same problem with
+`triage/duplicate` and close them. Only having a single issue to work on reduces confusion
+and avoids duplicate work on the same problem.
+
+### Dead link issues
+
+If the dead link issue is in the API or `kubectl` documentation, assign them `/priority critical-urgent` until the problem is fully understood. Assign all other dead link issues `/priority important-longterm`, as they must be manually fixed.
+
+### Blog issues
+
+We expect [Kubernetes Blog](https://kubernetes.io/blog/) entries to become
+outdated over time. Therefore, we only maintain blog entries less than a year old.
+If an issue is related to a blog entry that is more than one year old,
+it close the issue without fixing.
+
+### Support requests or code bug reports
+
+Some docs issues are actually issues with the underlying code, or requests for
+assistance when something, for example a tutorial, doesn't work.
+For issues unrelated to docs, close the issue with the `triage/support` label and a comment
+directing the requester to support venues (Slack, Stack Overflow) and, if
+relevant, the repository to file an issue for bugs with features (`kubernetes/kubernetes`
+is a great place to start).
+
+Sample response to a request for support:
+
+```none
+This issue sounds more like a request for support and less
+like an issue specifically for docs. I encourage you to bring
+your question to the `#kubernetes-users` channel in
+[Kubernetes slack](http://slack.k8s.io/). You can also search
+resources like
+[Stack Overflow](http://stackoverflow.com/questions/tagged/kubernetes)
+for answers to similar questions.
+
+You can also open issues for Kubernetes functionality in
+ https://github.com/kubernetes/kubernetes.
+
+If this is a documentation issue, please re-open this issue.
+```
+
+Sample code bug report response:
+
+```none
+This sounds more like an issue with the code than an issue with
+the documentation. Please open an issue at
+https://github.com/kubernetes/kubernetes/issues.
+
+If this is a documentation issue, please re-open this issue.
+```
+
+
+{{% /capture %}}
+
+
+{{% capture whatsnext %}}
+
```

Comment:
Is this intentionally empty? I _think_ `What's next` capture instr. are `optional` for concept templates.